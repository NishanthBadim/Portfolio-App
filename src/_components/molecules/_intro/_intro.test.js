import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Intro from "./_intro";

describe("Intro testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Intro/>);
    expect(input.toJSON()).toMatchSnapshot();
  })
})