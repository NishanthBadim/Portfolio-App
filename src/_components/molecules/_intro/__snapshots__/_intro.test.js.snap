// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro testing Matches the snapshot 1`] = `
<div>
  <h2
    style={
      Object {
        "paddingTop": "1.5em",
      }
    }
  >
    Nishanth Badim
  </h2>
  <h4
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Web Developer
  </h4>
  <hr
    style={
      Object {
        "borderTop": "3px solid #833fb2",
        "width": "60%",
      }
    }
  />
  <p>
    My personal philosophy when it comes to learning is closely related to several theories of motivation. I look things happening around me as simple as possible and postively and . I love designing websites and I wish my passion for web desinging continues.
  </p>
  <hr
    style={
      Object {
        "borderTop": "3px solid #833fb2",
        "width": "60%",
      }
    }
  />
  <h5>
    Address
  </h5>
  <p>
    Street No: 25, Hyderabad 500 062
  </p>
  <h5>
    Phone
  </h5>
  <p>
    (123) 456-7890
  </p>
  <h5>
    Email
  </h5>
  <p>
    nishanth.badim@zemosolabs.com
  </p>
  <hr
    style={
      Object {
        "borderTop": "3px solid #833fb2",
        "width": "50%",
      }
    }
  />
</div>
`;
